src/cli/commands/node.js

```diff
@@ -21,14 +21,16 @@ export function hasWrapper(commander: Object, args: Array<string>): boolean {
 export async function run(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<void> {
   const pnpPath = `${config.lockfileFolder}/${PNP_FILENAME}`;
 
+  let nodeOptions = process.env.NODE_OPTIONS || '';
   if (await fs.exists(pnpPath)) {
-    args = ['-r', pnpPath, ...args];
+    nodeOptions += ` --require ${pnpPath}`;
```

Comment:
I'm guessing the leading space wouldn't be an issue when `NODE_OPTIONS` is empty?

Reply:
Yep, will work 👍

Reply:
It looks like
```js
nodeOptions += ` --require ${pnpPath}`
```
should be 
```js
nodeOptions = `--require ${pnpPath} ${nodeOptions}`
```
so that pnp is the first preloaded module.